server/server.go

```diff
@@ -1365,3 +1366,32 @@ func (s *Server) reloadConfigFromKV() error {
 
 	return nil
 }
+
+// ReplicateFileToAllMembers is used to synchronize state among all members.
+// Each member will write `data` to a local file named `name`.
+func (s *Server) ReplicateFileToAllMembers(ctx context.Context, name string, data []byte) error {
+	resp, err := s.GetMembers(ctx, nil)
+	if err != nil {
+		return err
+	}
+	for _, member := range resp.Members {
+		url := member.GetClientUrls()[0] + filepath.Join("/pd/api/v1/admin/persist-file", name)
```

Comment:
Do we need to check the length of `GetClientUrls`?

Reply:
ok